GliaWidgets/Sources/CallVisualizer/VideoCall/ViewModel/VideoCallViewModel.swift

```diff
@@ -174,7 +174,7 @@ extension CallVisualizer.VideoCallViewModel {
                     closeButton: .init(style: style.header.closeButton),
                     endScreenshareButton: .init(
                         tap: .init { [weak self] in
-                            self?.environment.screenShareHandler.stop()
+                            self?.environment.screenShareHandler.stop(nil)
```

Comment:
I believe the stop parameter could be a little bit more descriptive. When you are using it by sending a callback, you are using the `stop { }` notation anyways so it doesn't matter what the name is. But in here, you just read `stop(nil)` and don't know what the `nil` is referring to.